entrypoint.sh

```diff
@@ -5,7 +5,7 @@ set -eu
 _main() {
     _switch_to_repository
 
-    if _git_is_dirty; then
+    if _git_is_dirty || [ "$INPUT_SKIP_DIRTY_CHECK" = true ]; then
```

Comment:
Wouldn't this be possible to simplify into the following? 
```sh
     if _git_is_dirty || "$INPUT_SKIP_DIRTY_CHECK"; then
```
As on Bash `[ "false" = false ]` is equal and `[ "true" = true ]` is equal also.

Reply:
@goetzc Yeah, that seems to work too. Thanks for the suggestion.